frontend/src/metabase/dashboard/components/DashboardGrid.jsx

```diff
@@ -224,10 +227,27 @@ export default class DashboardGrid extends Component {
     this.setState({ addSeriesModalDashCard: dc });
   }
 
+  getDashboardCardIcon = dashCard => {
```

Comment:
It may be cool to add some unit-test coverage to this component, even partial, related to what we're touching.

Reply:
Not sure it's possible until we upgrade unit-testing tooling :(